GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -25,5 +25,11 @@ public class LoggingBroker : ILoggingBroker
 
         public void LogWarning(string message) =>
             this.logger.LogWarning(message);
+            
+        public void LogError(Exception exception) =>
```

Comment:
Remove LogError. The scope of this PR is only to add LogCritical. When we merge your other PR this will be added